### ISSUE TITLE
Tabulator widget: `frozen_rows` not displayed

### ISSUE THREAD

**Issue by maximlt** [holoviz/panel#2955]
On `master` the `frozen_rows` list parameter actually hides the rows instead of displaying them at the top of the table view.

```python
import pandas as pd

import panel as pn

pn.extension('tabulator')

date_df = pd._testing.makeTimeDataFrame().iloc[:10]
agg_df = pd.concat([date_df, date_df.median().to_frame('Median').T, date_df.mean().to_frame('Mean').T])
agg_df.index= agg_df.index.map(str)

pn.widgets.Tabulator(agg_df, frozen_rows=[-2, -1], width=400)
```

![image](https://user-images.githubusercontent.com/35924738/143843621-bb408b31-6fbd-4672-8df5-cba7b0d67b33.png)

**Comment by philippjfr** [comment]
Partial fix in https://github.com/holoviz/panel/pull/2991

**Comment by philippjfr** [comment]
Fixed since reverting to tabulator 4.9.
